app/src/main/java/com/sanjeevyadavit/magecart/data/remote/ApiInterface.kt

```diff
@@ -11,19 +10,20 @@ import retrofit2.http.Query
 interface ApiInterface {
 
     @GET("V1/store/storeConfigs")
-    suspend fun storeConfigs(): List<StoreConfigs>
+    suspend fun storeConfigs(): List<StoreConfigsDto>
 
     @POST("V1/integration/customer/token")
-    suspend fun login(@Body loginBodyRequest: LoginBodyRequest): String
+    suspend fun login(@Body body: LoginBodyRequest): String
 
     @POST("V1/customers")
-    suspend fun signup(@Body signupBodyRequest: SignupBodyRequest): SignupResponse
+    suspend fun signup(@Body body: SignupBodyRequest): SignupDto
 
     @GET("V1/categories")
-    suspend fun getCategoryTree(): CategoryTree
+    suspend fun getCategoryTree(): CategoriesDto
 
     @GET("V1/products")
     suspend fun getProducts(
+        // QUESTION: Is there any better way to handle this, where you can send multiple filters
```

Comment:
I guess we need to have all the params that is needed for this api

Reply:
https://www.baeldung.com/retrofit-dynamic-url#:~:text=The%20Retrofit%20library%20allows%20us,be%20found%20over%20on%20GitHub